pages/datasets/index.tsx

```diff
@@ -4,20 +4,29 @@ import generateArcDataset from '../../lib/dataset/arcDataset'
 import generateBlobDataset from '../../lib/dataset/blobDataset'
 import generateConcentricRingsDataset from '../../lib/dataset/concentricRingsDataset'
 
-import Scatter from '../../lib/visualizations/svg/scatter'
+import Scatter from '../../lib/visualizations/d3/scatter'
+import Axis from '../../lib/visualizations/d3/axis'
+import SVGMultipleVisualization from '../../lib/visualizations/d3/svgmultiple'
 
 function Representation({ data, name, width, height }) {
     const svgRef = useRef(null)
 
     useEffect(() => {
-        const plot = new Scatter({ width, height })
+        const axisElemId = 'axis-elem'
```

Comment:
I now really think that we can use classes instead of ids - let me argument my decision
```
viz1:
  - #scatter-elem
  - #axis-elem

viz2:
  - #scatter-elem
  - #axis-elem
```
☝️ invalid HMTL DOM

vs.
```
viz1:
  - .scatter-elem
  - .axis-elem

viz2:
  - .scatter-elem
  - .axis-elem
```